configs/photo-sphere-viewer.json

```
@@ -34,12 +34,13 @@
     },
     "api": {
       "lvl0": {
-        "selector": ".main header h2",
+        "selector": "#noop",
```

```suggestion
        "selector": "",
```

This is how we usually do